.github/workflows/test-pg_search-stressgres.yml

```diff
@@ -7,11 +7,7 @@ name: Test pg_search Stressgres
 
 on:
   pull_request:
-    branches: # On the `staging` and `main` branches, we run Stressgres in PRs to test thoroughly
-      - main
-      - staging
-  push:
-    branches: # On the `dev` branch, we run Stressgres post pushes to speed up CI
+    branches: # On the default branch, we run Stressgres in PRs to test thoroughly
```

Comment:
The effect of this will be that we _don't_ run stressgres when a PR is merged: we should likely still do that, similar to the benchmark job.